### PR TITLE
updated visibility of release_builds

### PR DESCRIPTION
**Pull request by LambdaLearner**
https://dimagi-dev.atlassian.net/browse/ICDS-1531 Updated visibility of Manage releases

### CODE REVIEW

**Review by mkangia — CHANGES_REQUESTED** [review]
@AmitPhulera can you guide Sanchit on how we talked about implementing this. The original problem is that the concerned user does not even have access to the Projects tab. I think we talked about placing this link on profiles tab on the application page.

**Comment by AmitPhulera** [comment]
Yeah sure @mkangia.
Hey @dungeonmaster51 let's offline on it.

File: corehq/apps/app_manager/templates/app_manager/partials/releases/build_profiles.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 {% load hq_shared_tags %}
+<a href="{% url 'manage_releases_by_app_profile' domain %}">Manage Releases By App Profile</a>
```
Comment (mkangia) [inline corehq/apps/app_manager/templates/app_manager/partials/releases/build_profiles.html:3]: this is just adding the link, we would also need to check if we need to show it or not based on permissions.

**Review by mkangia — CHANGES_REQUESTED** [review]

File: corehq/apps/app_manager/templates/app_manager/partials/releases/build_profiles.html (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/app_manager/templates/app_manager/partials/releases/build_profiles.html:4]: Link text needs to go through `trans`, please.

**Comment by orangejenny** [comment]
Hi Sanchit, this is a small change but I'm curious what it looks like in the UI. Could you add a screenshot?

**Comment by snopoke** [comment]
Closing old PRs. Reopen if necessary.

**Pull request closed without merging**